src/arm/96boards.c

```diff
@@ -47,12 +47,21 @@ int db410c_ls_gpio_pins[MRAA_96BOARDS_LS_GPIO_COUNT] = {
     36, 12, 13, 69, 115, 4, 24, 25, 35, 34, 28, 33,
 };
 
+int db410c_chardev_map[MRAA_96BOARDS_LS_GPIO_COUNT][2] = {
+    { 0, 36 }, { 0, 12 }, { 0, 13 }, { 0, 69 }, { 0, 115 }, { 0, 129 },
```

Comment:
Does line 129 map to gpio4? Everything else besides this is identical.

Reply:
That specific pin comes from PMIC, hence I assumed that it belongs to the same gpiochip. But further investigation reveals that it belongs to Pin 3 of gpiochip 2. Will change it.